samples/web/content/apprtc/analytics.py

```diff
@@ -0,0 +1,91 @@
+import constants
+import datetime
+import httplib2
+import json
+import logging
+import os
+import time
+import webapp2
+
+from apiclient import discovery
+from oauth2client.appengine import AppAssertionCredentials
+from oauth2client.client import SignedJwtAssertionCredentials
+
+BIGQUERY_URL='https://www.googleapis.com/auth/bigquery'
+
+class Analytics(object):
+  """Class used to encapsulate analytics logic.
+
+  All data is streamed to BigQuery. Logging to BigQuery when running
+  locally requires a 'secrets.json' containing Service Account
+  credentials.
+
+"""
+  def __init__(self):
+    is_running_locally = os.environ['APPLICATION_ID'].startswith('dev')
+
+    self.bigquery_table = 'analytics'
+    self.bigquery_project = 'valiant-broker-834'
+
+    if is_running_locally:
+      self.bigquery_dataset = 'dev'
+    else:
+      self.bigquery_dataset = 'prod'
```

Comment:
We may want some way to differentiate the data from different server versions, i.e. 1-dot-apprtc, 3-dot-apprtc. Can we pass that from the caller and use different dataset?